csun-metalab/nomi
# NAM-552 -- UX: Profile images in bottom navigation and side menu need alt text

# Description
Added alt-text for all `profile image's` .

# Testing
1. Login as `steve` . 
2. Ensure all profile images contain a descriptive `alt text` using your inspector.
